.github/workflows/test.yml

```diff
@@ -24,7 +24,10 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install -y man2html-base pycodestyle python3-bs4 python3-markdown
+          sudo apt-get install -y man2html-base pycodestyle python3-bs4
+
+      - name: Install Python dependencies
+        run: pip3 install -r requirements.txt
```

Comment:
@ru-fu I think you may need to run that under `sudo` so it's globally available for the next stage.

Reply:
Thanks! That saved me a lot of debugging. :+1: